README.md

```diff
@@ -1 +1,56 @@
-# gene2phenotype_api
\ No newline at end of file
+# Gene2Phenotype (G2P) API
+
+The Gene2Phenotype API provides a RESTful service to access evidence-based gene-disease models, curated from the literature by experts.
+It is part of the Gene2Phenotype project, established by David FitzPatrick in 2012 with the aim of accelerating the diagnosis of children with developmental disorders. In 2014, the database moved from the University of Edinburgh to EMBL-EBI and a dedicated website was launched to improve data accessibility. G2P has been generalised to cover other disease areas and extended to capture additional information to provide a more detailed understanding of disease mechanism. To read more about the project please visit the [G2P website](https://www.ebi.ac.uk/gene2phenotype/about/project).
+
+API documentation is available at [Gene2Phenotype API Documentation](https://www.ebi.ac.uk/gene2phenotype/api/).
+
+### Installation and requirements
```

Comment:
We already talked about this but i do not think we should have the installation here, Requirement is fine.